flake.nix

```diff
@@ -11,29 +11,39 @@
 
     nixpkgs.url = "nixpkgs";
 
+    yants.url = "github:divnix/yants";
+
     dream2nix = {
       url = "github:nix-community/dream2nix";
       inputs.nixpkgs.follows = "nixpkgs";
     };
   };
 
   outputs = {
-      std,
-      self,
-      ...
+    std,
+    self,
+    ...
   } @ inputs:
     std.growOn {
       inherit inputs;
       cellsFrom = ./cells;
       cellBlocks = with std.blockTypes; [
+        # app
         (installables "packages" {ci.build = true;})
         (devshells "devshells")
         (functions "dream2nix")
         (nixago "configs")
+
+        # lib
+        ## for external consumption
+        (functions "nixt")
+        ## for cli use
+        (functions "standalone")
       ];
     }
-      {
-        packages = std.harvest self ["nixt" "packages"];
-        devShells = std.harvest self ["nixt" "devshells"];
-      };
+    {
+      packages = std.harvest self ["app" "packages"];
+      devShells = std.harvest self ["app" "devshells"];
+      lib = std.harvest self ["lib" "nixt"];
```

Comment:
If you also want to expose `lib.nixt.inject` in addition to `lib.inject` (for whatever reason; maybe you don't).

Then, there's a trick to do that which goes like this:
`lib = std.harvest self [ ["lib"] ["lib" "nixt"]]`

In cases where the lib is system agnostic, which it looks like it _is_, you can use `std.pick`, instead.

Reply:
`std.pick` is now used. Is there any value in exposing lib.nix.inject? It's mainly intended for CLI use.

Reply:
> value in exposing lib.nix.inject?

I don't think so. But I took the opportunity to spread the word about that little trick. :smile: